src/plugins/maven-resolver.js

```diff
@@ -0,0 +1,134 @@
+const Joi = require('joi');
+const got = require('got');
+const findReachableUrls = require('find-reachable-urls');
+const parseString = require('xml2js').parseString;
+const Boom = require('boom');
+const insight = require('../utils/insight');
+
+function notFoundResponse() {
+  return Boom.notFound('Package not found', {
+    eventKey: 'package_not_found',
+  });
+}
+
+const praseXML = async str => new Promise((resolve, reject) => {
+  parseString(str, (err, result) => {
+    if (err) {
+      return reject(err);
+    }
+
+    resolve(result);
+  });
+});
+
+const getMavenResponse = async (namespace) => {
+  const requestUrl = `https://search.maven.org/solrsearch/select?wt=json&rows=1&q=fc:%22${namespace}%22`;
```

Comment:
The response is changing from time to time and therefore not reliable useful for us

Reply:
Note to myself: check out http://grepcode.com/api